e2e_test/e2e_test.py

```diff
@@ -47,4 +47,14 @@ def test_check_title(driver, test_app):
 #     button = driver.find_element_by_xpath("add-button")
 #     button.click()
     
-#     assert text in driver.find_element_by_xpath("//ul").text
\ No newline at end of file
+#     assert text in driver.find_element_by_xpath("//ul").text
+
+
+@pytest.fixture(scope='module')
+def driver():
```

Comment:
You should delete the Firefox version of this above and update your README now that the tests depend on Chrome instead